dockers/docker-orchagent/orchagent.sh

```diff
@@ -2,9 +2,10 @@
 
 # Export platform information. Required to be able to write
 # vendor specific code.
-export platform=`sonic-cfggen -y /etc/sonic/sonic_version.yml -v asic_type`
+SWSS_VARS=$(sonic-cfggen -d -y /etc/sonic/sonic_version.yml -t /usr/share/sonic/templates/swss.j2)
+export platform=$(echo $SWSS_VARS | jq -r '.asic_type')
```

Comment:
>export [](start = 0, length = 6)

If removing `export`, is it the same?

Reply:
should be, not sure why the original script had export!